main.js

```diff
@@ -0,0 +1,79 @@
+function myFunction() {
+  alert("Page is loaded");
+}
+
+$.getJSON("data/sample.json", function(json) {
+  let basics = [];
+  let work = [];
+  for (const eachItem in json.basics) {
+    if (json.basics.hasOwnProperty(eachItem)) {
+      basics.push(json.basics[eachItem]);
+    }
+  }
+
+  console.log(json.work);
+
+  //Building basics HTML:
+  $(".navName").html(basics[0]);
+  $(".nameOnly").html(basics[0]);
+  $(".address").html(
+    basics[7]["address"] +
+      " · " +
+      basics[7]["postalCode"] +
+      " · " +
+      basics[7]["city"] +
+      " · " +
+      basics[7]["region"] +
+      " · " +
+      basics[7]["countryCode"]
+  );
+  $(".email").html(basics[3]); //Cannot access HTML elements that are inside other HTML elements.
+  $(".aboutMe").html(basics[6]);
+
+  //Building Experience HTML:
+  //Ideally what I what is to build each HTML element based on each work object and
+  //then use those individual HTML elements to build a div which I then add to document.
+  //But I ran out of time trying to figure out jQuery
+
+  let divTitle, divDesc;
+  json.work.forEach(function(d) {
+    console.log(d);
+    divTitle = document.createElement("h3");
+    $(divTitle).html(d["position"]);
+
+    // $(".resume-title").html(d["position"]);
+  });
+
+  let divContainer = document.getElementById("divResume");
+  divContainer.innerHTML = "";
+  divContainer.appendChild(divTitle);
+
+  // // CREATE DYNAMIC TABLE.
+  // var table = document.createElement("table");
+
+  // // CREATE HTML TABLE HEADER ROW USING THE EXTRACTED HEADERS ABOVE.
+
+  // var tr = table.insertRow(-1);                   // TABLE ROW.
+
+  // for (var i = 0; i < col.length; i++) {
+  //     var th = document.createElement("th");      // TABLE HEADER.
+  //     th.innerHTML = col[i];
+  //     tr.appendChild(th);
+  // }
+
+  // // ADD JSON DATA TO THE TABLE AS ROWS.
+  // for (var i = 0; i < myBooks.length; i++) {
+
+  //     tr = table.insertRow(-1);
+
+  //     for (var j = 0; j < col.length; j++) {
+  //         var tabCell = tr.insertCell(-1);
+  //         tabCell.innerHTML = myBooks[i][col[j]];
+  //     }
+  // }
+
+  // // FINALLY ADD THE NEWLY CREATED TABLE WITH JSON DATA TO A CONTAINER.
+  // var divContainer = document.getElementById("showData");
+  // divContainer.innerHTML = "";
+  // divContainer.appendChild(table);
+});
```

Comment:
I don't think I can add anything constructive in terms of the code as I am super new to the game. 
Loved the colours and layout of the deployed site.
What I may suggest is to adopt an MVP approach next time.  I couldn't see much of the JSON file displayed on the deployed site.  Basically, get a functioning site up and running first albeit ugly, then worry about styling later. (In my noob opinion)

Reply:
I agree with your MVP statement.  I am very guilty of this myself, even with this project.  Ran out of time because I kept getting lost in the other stuff I shouldnt had been worrying about yet.

Reply:
I agree with you. There are two schools of thought. You either work on one part at a time and get it looking right before moving to the next. ie. like working on Basic first. 
Doing this is hard in hackathons because you have limited time and you don't want to deliver an unfinished product. 

I've adapted to getting the data on at first and working on styling later. So you have somewhat of a finished product to show. Though you'll see in my project I didn't get further onto styling lol... so I was also incomplete.

Reply:
Otherwise, I do not really have much to add.  I think you were off to a great start.  For what you have on your deployed site, it looks great.  With some more time, I think you would of had something really beautiful there.  I like your structuring, and css layout (only thing maybe I'd suggest would just be look into using Sass, I never build without it now, just for the nested elements and use of variables).  Great work!